understanding/focus-appearance.html

```diff
@@ -445,498 +372,379 @@ <h4>Inline links</h4>
                </section>
                <section id="change-of-contrast">
                   
-                  <h3>Change of contrast</h3>
+                  <h3>コントラストの変化</h3>
                   
                   
-                  <p>The second part of the Success Criterion's indicator requirements states that an area
-                     of the indicator:
+                  <p>達成基準のインジケータ要件の 2 番目の部分では、インジケータの領域について次のように述べている:
                   </p>
                   
                   
                   <blockquote>
                      
                      <ul>
                         
-                        <li>has a contrast ratio of at least 3:1 between the same pixels in the focused and unfocused
-                           states
+                        <li>フォーカスされている状態とフォーカスされていない状態との間で、同一ピクセルに少なくとも 3:1 のコントラスト比がある。
                         </li>
                         
                      </ul>
                      
                   </blockquote>
                   
                   
-                  <p>This requirement measures the change of contrast between the same pixels in different
-                     states. This is different from the Text Contrast and Non-text Contrast Success Criteria,
-                     which measure the contrast between different adjacent pixels in a single state at
-                     a time.
+                  <p>この要件は、異なる状態における同一ピクセル間のコントラストの変化を測定する。これは、一度に単一状態における隣接する異なるピクセル間のコントラストを測定するテキストのコントラスト及び非テキストのコントラストの達成基準とは異なる。
```

Comment:
「一度に単一状態における (in a single state at a time.)」がわかりにくい感じがしたので、少し手を加えてみました。

```suggestion
                  <p>この要件は、異なる状態における同一ピクセル間のコントラストの変化を測定する。これは、一時のある一つの状態において隣接する異なるピクセル間のコントラストを測定する、テキストのコントラスト及び非テキストのコントラストの達成基準とは異なる。
```

Reply:
@caztcha 

at a timeを「同時に」とするのでどうでしょう？

```suggestion
                  <p>この要件は、異なる状態における同一ピクセル間のコントラストの変化を測定する。これは、同時にある一つの状態において、隣接する異なるピクセル間のコントラストを測定する、テキストのコントラスト及び非テキストのコントラストの達成基準とは異なる。
```

Reply:
@momdo ありがとうございます。よいとおもいます！（いただいた提案にて、commit いたしました。）